visualizer/Readme.md

```diff
@@ -15,36 +16,45 @@ Certains kind of edits, like editing the NetBeans module exlusions or editing th
 components, will require using NetBeans.
 
 There is a known issue with opening the Maven based project in NetBeans because of a bug with the
-`mx` project support built into NetBeans itself.`  Because 
+`mx` project support built into NetBeans itself.  The `mx` support seems to interfere with
+recognizing the maven modules so at the moment it's necessary to disable the `java-mx-support`
+module in NetBeans 22.  On the mac this can be done by editing the file `$HOME/Library/Application
```

Comment:
Have you tried the `.mxignore` file?

Reply:
As far as I can say:
```
/graalvm/graal/visualizer/IdealGraphVisualizer$ echo >.mxignore
/graalvm/graal/visualizer/IdealGraphVisualizer$ netbeans-22/bin/netbeans --open .
```
Fixes the problem and I can open the Maven pom projects as all its children.

Reply:
Thanks for figuring this out.  I'll add it to the repo and update the readme.